apps/client/src/login/LoginForm.test.tsx

```diff
@@ -0,0 +1,53 @@
+import { screen, render, fireEvent } from '@testing-library/react';
+
+import renderRouter from '../testing/routes';
+import LoginForm from './LoginForm';
+
+const setup = () => {
+  const handleSubmit = jest.fn();
+  const utils = render(<LoginForm handleSubmit={handleSubmit} />);
+  const userName = screen.getByRole('textbox', {
+    name: /userName/i,
+  });
+
+  const password = screen.getByRole('textbox', {
+    name: /password/i,
+  });
+
+  const submitButton = screen.getByRole('button', {
+    name: /Submit/i,
+  });
+
+  return {
+    userName,
+    password,
+    submitButton,
+    handleSubmit,
+    ...utils,
+  };
+};
+
+it('updates the inputs', () => {
+  const { userName, password } = setup();
+  renderRouter('/');
+
+  fireEvent.change(userName, { target: { value: 'user' } });
+  fireEvent.change(password, { target: { value: 'password' } });
+
+  expect((userName as HTMLInputElement).value).toBe('user');
+  expect((password as HTMLInputElement).value).toBe('password');
```

Comment:
Not blocking: Could we cast the return value inside of [`setup`](https://github.com/awslabs/iot-application/pull/42/files#diff-4d36cd9199f1e3df8e379a88f735a7f352c524a2c8c88cdac469baa1cc5c8a08R6) instead to reduce casting in the tests as the suite grows?